.github/workflows/labeler.yml

```
@@ -26,19 +29,12 @@ jobs:
             "members": "read"
           }
     - name: Add agent-python label
-      uses: actions-ecosystem/action-add-labels@v1
-      with:
-        labels: agent-python
+      run: gh issue edit "$NUMBER" --add-label "agent-python"
     - id: is_elastic_member
       uses: elastic/oblt-actions/github/is-member-of@v1
       with:
         github-org: "elastic"
         github-user: ${{ github.actor }}
         github-token: ${{ steps.get_token.outputs.token }}
     - name: Add community and triage labels
-      if: contains(steps.is_elastic_member.outputs.result, 'false') && github.actor != 'dependabot[bot]' && github.actor != 'elastic-observability-automation[bot]'
-      uses: actions-ecosystem/action-add-labels@v1
-      with:
-        labels: |
-          community
-          triage
+      run: gh issue edit "$NUMBER" --add-label "community,triage"
```

```bash
$ gh issue edit "2257" --add-label "community,triage"  
https://github.com/elastic/apm-agent-python/pull/2257
```

---

What about the if?

---

I accidentally dropped it :/

```suggestion
      if: contains(steps.is_elastic_member.outputs.result, 'false') && github.actor != 'dependabot[bot]' && github.actor != 'elastic-observability-automation[bot]'
      run: gh issue edit "$NUMBER" --add-label "community,triage"
```